membership_variable_period/models/product_product.py

```diff
@@ -0,0 +1,14 @@
+# -*- coding: utf-8 -*-
+# Copyright 2016 Antonio Espinosa <antonio.espinosa@tecnativa.com>
+# License AGPL-3 - See http://www.gnu.org/licenses/agpl-3.0.html
+
+from openerp import models, api
+
+
+class ProductProduct(models.Model):
+    _inherit = "product.product"
+
+    @api.multi
+    def _get_next_date(self, date, qty=1):
```

Comment:
Well, this may apply for fields but not for methods.
I've tested removing the method and tests crash with this error:

```
2016-11-17 17:11:02,407 15150 ERROR variable_period openerp.addons.membership_prorrate_variable_period.tests.test_membership_prorrate_variable_period: ERROR: test_get_next_date (openerp.addons.membership_prorrate_variable_period.tests.test_membership_prorrate_variable_period.TestMembershipProrrateVariablePeriod)
2016-11-17 17:11:02,407 15150 ERROR variable_period openerp.addons.membership_prorrate_variable_period.tests.test_membership_prorrate_variable_period: Traceback (most recent call last):
2016-11-17 17:11:02,407 15150 ERROR variable_period openerp.addons.membership_prorrate_variable_period.tests.test_membership_prorrate_variable_period: `   File "/opt/odoo/v8/other-addons/membership_prorrate_variable_period/tests/test_membership_prorrate_variable_period.py", line 115, in test_get_next_date
2016-11-17 17:11:02,407 15150 ERROR variable_period openerp.addons.membership_prorrate_variable_period.tests.test_membership_prorrate_variable_period: `     self.product._get_next_date(date))
2016-11-17 17:11:02,407 15150 ERROR variable_period openerp.addons.membership_prorrate_variable_period.tests.test_membership_prorrate_variable_period: ` AttributeError: 'product.product' object has no attribute '_get_next_date'
2016-11-17 17:11:02,407 15150 INFO variable_period openerp.addons.membership_prorrate_variable_period.tests.test_membership_prorrate_variable_period: Ran 7 tests in 2.021s
2016-11-17 17:11:02,407 15150 ERROR variable_period openerp.addons.membership_prorrate_variable_period.tests.test_membership_prorrate_variable_period: FAILED
2016-11-17 17:11:02,407 15150 INFO variable_period openerp.addons.membership_prorrate_variable_period.tests.test_membership_prorrate_variable_period:  (errors=5)

```